src/planet-a/IcoSphereTree.h

```diff
@@ -33,14 +38,23 @@ namespace planeta
 
 // Index to a triangle
 using trindex_t = uint32_t;
+constexpr trindex_t gc_invalidTri = std::numeric_limits<trindex_t>::max();
```

Comment:
I think that constexpr automatically implies the inline keyword, but you might want to double check that you don't want the inline keyword as well.